styles/components/general/GeneralLogo.module.sass

```
@@ -0,0 +1,19 @@
+@use '@/styles/utils/mixins' as m
+@use '@/styles/utils/funcs' as f
+@use '@/styles/themes/default/Colors.module.sass' as c
+
+.container 
+    display: flex
+    align-items: flex-end
+
+.icon
+    width: f.px(47)
```

should be replaced to max-width
```suggestion
    max-width: f.px(47)
```